### PR TITLE
Update course assignments endpoint

### LINKED ISSUE #23: API documentation for /course/:id/assignments is incomplete
API documentation for `/course/:id/assignments` is incomplete. The docs say:

```json
    {
      "id": 2,
      "resource_type": "assignment",
      "assignee_id": 2,
      "ext_uid": "DEF456",
      "due_by": "2020-09-30T00:00:00Z",
      "reassigned_at": "2020-09-30T00:00:00Z"
    }
```

An actual response:

```json
        {
            "id": 977127,
            "resource_type": "assignment",
            "assignee_id": 564601,
            "assignable_type": "Course",
            "assignable_id": 2722,
            "ext_uid": null,
            "due_by": null,
            "reassigned_at": null,
            "status": "Completed",
            "score": 0,
            "started_at": "2016-03-28T14:15:17Z",
            "completed_at": "2016-03-28T18:20:06Z",
            "updated_at": "2016-03-28T18:20:06Z"
        }
```

### PR DESCRIPTION
**Pull request by armilam**
## Why?

Fixes #23 

The API docs were missing some info about course assignments.

## What?

Add assignable_type, assignable_id, status, started_at, completed_at, and updated_at to documentation for course assignments endpoint.

### CODE REVIEW

File: source/includes/_courses.md [hunk 1]
```diff
@@ -93,17 +93,29 @@ curl -u "DOMAIN:API_KEY" "https://api.lesson.ly/api/v1/courses/:course_id/assign
       "id": 1,
       "resource_type": "assignment",
       "assignee_id": 1,
+      "assignable_type": "Course",
+      "assignable_id": 1,
       "ext_uid": "ABC123",
       "due_by": "2020-09-30T00:00:00Z",
-      "reassigned_at": "2020-09-30T00:00:00Z"
+      "reassigned_at": "2020-09-30T00:00:00Z",
```
Comment (stevegrossi) [inline source/includes/_courses.md]: I smell a prophesy...

File: source/includes/_courses.md [hunk 2]
```diff
@@ -93,17 +93,29 @@ curl -u "DOMAIN:API_KEY" "https://api.lesson.ly/api/v1/courses/:course_id/assign
...
+      "assignable_type": "Course",
+      "assignable_id": 1,
       "ext_uid": "DEF456",
       "due_by": "2020-09-30T00:00:00Z",
-      "reassigned_at": "2020-09-30T00:00:00Z"
+      "reassigned_at": "2020-09-30T00:00:00Z",
+      "status": "Incomplete",
+      "started_at": "2016-03-28T14:15:17Z",
+      "completed_at": null,
+      "updated_at": "2016-03-28T18:20:06Z"
```
Comment (stevegrossi) [inline source/includes/_courses.md]: It looks like we technically (though I suspect incorrectly) send a `"score"` for course assignments as well, but it's always `0` (which is misleading):

screen shot 2016-04-15 at 1 16 38 pm

I'd say let's stop doing that, but that would be a breaking change to the API which some consumers could be depending on (i.e. expecting all assignments to have a `"score"` attribute) so I guess we need to document that here as well, eh?

Comment (armilam) [inline source/includes/_courses.md]: I say why document it if it's not really a feature? If it's not something we want people to rely on? Technically, it wouldn't be a breaking change for us to remove something that was never meant to be there and we never told anybody about it.

My opinion on that, anyway. What do you think, @stevegrossi?

Comment (stevegrossi) [inline source/includes/_courses.md]: Yeah, I suppose if we never documented it's not really a breaking change to remove it. Good call, I'll merge this and create an issue to stop sending `score: 0` for all the course assignments.

**Comment by stevegrossi** [comment]
Looks good—thanks for adding this! I'll merge and publish!

**Pull request merged**